src/gantt.ts

```diff
@@ -843,17 +901,23 @@ export class Gantt implements IVisual {
      * @param isEndDateFilled
      * @param hasHighlights if any of the tasks has highlights
      */
-    private static createTasks(
-        dataView: DataView,
-        taskTypes: TaskTypes,
-        host: IVisualHost,
-        formatters: GanttChartFormatters,
-        colors: IColorPalette,
-        settings: GanttChartSettingsModel,
-        taskColor: string,
-        localizationManager: ILocalizationManager,
-        isEndDateFilled: boolean,
-        hasHighlights: boolean): Task[] {
+    private createTasks({
+        dataView,
+        taskTypes,
+        formatters,
+        taskColor,
+        isEndDateFilled,
+        hasHighlights,
+        sortingOptions
+    }: {
```

Comment:
Let's create and use interfaces when we have such a large object as an argument. (You can create a separate folder/file for them and import where it is needed, even if it is required just once. That will make the main code logic more lightweight and easier to read.)